### PR TITLE
chore: disable kotlin example

### PR DESCRIPTION
**Pull request by alexeagle**
It's broken by an internet outage, see #2747

### CODE REVIEW

File: examples/kotlin/BUILD.bazel [hunk 1]
```diff
@@ -11,8 +11,8 @@ load("@npm//http-server:index.bzl", "http_server")
 # Grab a Maven dependency
 kt_js_import(
     name = "kotlinx-html-js",
-    jars = ["@maven//:v1/http/dl.bintray.com/kotlin/kotlinx.html/org/jetbrains/kotlinx/kotlinx-html-js/0.6.12/kotlinx-html-js-0.6.12.jar"],
-    srcjar = "@maven//:v1/http/dl.bintray.com/kotlin/kotlinx.html/org/jetbrains/kotlinx/kotlinx-html-js/0.6.12/kotlinx-html-js-0.6.12-sources.jar",
+    jars = ["@maven//:v1/https/repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-html-js/0.7.3/kotlinx-html-js-0.7.3.jar"],
```
Comment (mattem) [inline examples/kotlin/BUILD.bazel:14]: Does 0.6.13 work? https://dl.bintray.com/kotlin/kotlin-dev/org/jetbrains/kotlinx/kotlinx-html-js/0.6.13-build-1696/kotlinx-html-js-0.6.13-build-1696.jar

Comment (alexeagle) [reply examples/kotlin/BUILD.bazel:14]: I didn't realize that any bintray URLs still worked... the registry doesn't respond so you can't do the `bazel run @unpinned_maven//:pin` with any bintray urls

**Review by mattem — APPROVED** [review]

**Comment by alexeagle** [comment]
Somehow this is working again on CI, but I'll keep this around. I suspect bintray is up-and-down but will soon be permanently down.

**Comment by alexeagle** [comment]
This is breaking again so let's land it

**Pull request merged**